### ISSUE TITLE
minigrad ~0 loss on cross entropy (mismatch to pytorch)

### ISSUE THREAD

**Issue by tostacam** [tostacam/minigrad#4]
Loss mismatch: [minigrad: -1.850371707708594e-17, pytorch: 0.8820791499159654]
